.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,57 @@
+name: Github Actions
+
+on: [push, pull_request]
+
+jobs:
+  test_x86:
+    runs-on: ubuntu-latest
```

Comment:
Shall we specify the version for Ubuntu Linux? That is, 20.04-LTS in particular.

Reply:
The `ubuntu-latest` would be `18.04` now.
Please check:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

Besides, the Github Actions page has warned us that:
```
Ubuntu-latest workflows will use Ubuntu-20.04 soon.
For more details, see https://github.com/actions/virtual-environments/issues/1816 
```
I think we should replace it with `20.04`.
Based on the https://github.com/actions/virtual-environments/issues/1816, it would use `Clang` version 10 as default, which is the compiler version we use.
Soon the `ubuntu-latest` would become `20.04`.